lib/clearance/session.rb

```diff
@@ -112,9 +106,27 @@ def cookies
       @cookies ||= ActionDispatch::Request.new(@env).cookie_jar
     end
 
+    # @api private
+    def set_remember_token(token)
+      case Clearance.configuration.signed_cookie
+      when true, :migrate
+        cookies.signed[remember_token_cookie] = cookie_options(token)
+      when false
+        cookies[remember_token_cookie] = cookie_options(token)
```

Comment:
I'm curious what the difference with this new implementation. We used to use `Rack::Utils` Is there any meaningful difference here?

Reply:
As far as I understand, `Rack::Utils` is probably much lower level. Rails might use it under the hood, or use some other way to add cookies to Rack. Since Clearance is geared towards Rails (exclusively?), then I think it makes sense to use the Rails' built-in helper for handling cookies rather than `Rack::Utils`... I could be wrong though, I'm not that familiar with Rails internals and didn't dig too deep into it.